docs/modules/ROOT/pages/usage/lsp.adoc

```diff
@@ -131,6 +131,30 @@ For detailed instructions on setting the parameter, please refer to the configur
 
 NOTE: The `safeAutocorrect` parameter was introduced in RuboCop 1.54.
 
+== Lint Mode
+
+LSP client can run lint cops by passing the following `lintMode` parameter in the `initialize` request
+if you only want to enable the feature as a linter like `ruby -w`:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 42,
+  "method": "initialize",
+  "params": {
+    "initializationOptions": {
+      "lintMode": true
+    }
+  }
+}
+```
+
+Furthermore, enabling autocorrect in a LSP client at the time of saving equates to the effect of `rubocop -l` option.
+
+For detailed instructions on setting the parameter, please refer to the configuration methods of your LSP client.
+
+NOTE: The `layoutMode` parameter was introduced in RuboCop 1.55.
```

Comment:
I think you meant to write lintMode here.

Reply:
Oops! I've updated this PR.